File: Java/Introduction/src/ThreeNumbers.java
/*Write a program that accepts three numbers and computes their sum, product and average. */
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class ThreeNumbers {
    public static void main(String[] args) throws IOException {
        int number_one, number_two, number_three, sum, product;
        float average;

        InputStreamReader isr = new InputStreamReader(System.in);
        BufferedReader br = new BufferedReader(isr);

        System.out.println("Enter the first number");
        number_one=Integer.parseInt(br.readLine());
        System.out.println("Enter the second number");
        number_two=Integer.parseInt(br.readLine());
        System.out.println("Enter the third number");
        number_three=Integer.parseInt(br.readLine());

        sum=number_one+number_two+number_three;
        product=number_one*number_two*number_three;
        average=sum/3;

        System.out.println("The sum is "+sum+", product is "+product+" and average is "+average);

    }
}

File: C++/Introduction/Three Numbers.cpp
/*Write a program that accepts three numbers and computes their sum, product and average. */
#include <iostream>
using namespace std;
int main()
{
    int num1, num2, num3, sum, product;
    float division, average;
    cout<<"Enter the integers\n";
    cin>>num1>>num2>>num3;
    sum=num1+num2+num3;
    product=num1*num2*num3;
    average=sum/3;
    cout<<"The sum is "<<sum;
    cout<<"\nThe product is "<<product;
    cout<<"\nThe average is "<<average;
    return 0;
}
